### DISCUSSION TITLE
Confusion in Installing Bullet for GNU/Linux

### DISCUSSION THREAD

**Discussion by glanzkaiser** [bulletphysics/bullet3#4255] [Q&A]
Dear all,

I want to make sure if I installed Bullet correctly,

I git clone the repository then

From the github' readme guidance, I run this inside the Bullet folder:
**./build_cmake_pybullet_double.sh**

Then I move the build_cmake folder to my local directory I have succesfully run the AppBrowser

This is my question, I actually tried another method mentioned, I try the second option to install Bullet with premake :

**cd build3
./premake4_linux64 --double gmake**

I get a different result it becomes a folder inside the bullet3-3.24/build3/gmake/obj/x64/Release, contain folders of examples with .d and .o files
I do not know how to use this and proceed. 
This is the screenshot from premake result:
https://ibb.co/2Z1zxxj

I also want to know about the /src folder in Bullet repo:
**https://github.com/bulletphysics/bullet3/tree/master/src**
can this be moved directly to our include folders? I can create environment variables to this if it is possible.

**Comment by CrezyDud** [comment]
bruh
1st: its not "GNU/Linux"
2nd: look at the README its right Fucing there you used the wrong command
